[0m09:40:17.706865 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001202DF6A510>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001202AF09BD0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001202E85D450>]}


============================== 09:40:17.745799 | 07e9dadc-4cd8-4ed3-a28d-3ddc081f63a2 ==============================
[0m09:40:17.745799 [info ] [MainThread]: Running with dbt=1.9.6
[0m09:40:17.746612 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'fail_fast': 'False', 'debug': 'False', 'log_path': 'logs', 'profiles_dir': 'C:\\Users\\Micha\\.dbt', 'version_check': 'True', 'use_colors': 'True', 'use_experimental_parser': 'False', 'empty': 'None', 'quiet': 'False', 'no_print': 'None', 'log_format': 'default', 'invocation_command': 'dbt init DataEng01', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'introspect': 'True', 'target_path': 'None', 'static_parser': 'True', 'send_anonymous_usage_stats': 'True'}
[0m09:40:17.875250 [debug] [MainThread]: Starter project path: D:\dbt_env\Lib\site-packages\dbt\include\starter_project
[0m09:40:17.889921 [info ] [MainThread]: 
Your new dbt project "DataEng01" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m09:40:17.890826 [info ] [MainThread]: Setting up your profile.
[0m09:40:41.613142 [info ] [MainThread]: Profile DataEng01 written to C:\Users\Micha\.dbt\profiles.yml using target's profile_template.yml and your supplied values. Run 'dbt debug' to validate the connection.
[0m09:40:41.614208 [debug] [MainThread]: Command `dbt init` succeeded at 09:40:41.614055 after 24.04 seconds
[0m09:40:41.614505 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001202ED9B100>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001202EDC7D10>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001202EB7F240>]}
[0m09:40:41.614797 [debug] [MainThread]: Flushing usage events
[0m09:40:43.708389 [debug] [MainThread]: An error was encountered while trying to flush usage events
